packages/react-next/src/components/Checkbox/Checkbox.scss

```diff
@@ -0,0 +1,253 @@
+@import '../../common/themeCssVariables';
```

Comment:
I think this pattern would write the variables to every individual component, leading to bulky downloads.

A better approach would be to define sass variables for the names themselves and import that.

_ThemeVariables.scss:

```sass
@semanticColorsInputBorderHovered: --semanticColorsInputBorderHovered;
@semanticColorsInputBorderHovered: --semanticColorsInputBorderHovered;
```

Checkbox.scss:

```sass
background:  var(@semanticColorsInputBorderHovered);

// Variable is not declared, should break build.
background:  var(@semanticColorsInputBorderHoveredTypo);
```

Reply:
yeah like you said , this is temp to just unblock developing the styles. 
Good call on assign the css var to as sass var. i didn't think of that idea!